SomosMasApp/SomosMasApp/App/Shared/Modules/Home/Controller/TestimoniosCollectionViewCell.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  TestimoniosCollectionViewCell.swift
+//  SomosMasApp
+//
+//  Created by Fran Nasich on 01/04/2022.
+//
+
+import UIKit
+
+class TestimoniosCollectionViewCell: UICollectionViewCell {
+
+    @IBOutlet weak var testimonioImage: UIImageView!
+    @IBOutlet weak var testimonioEpigrafe: UILabel!
+    @IBOutlet weak var verMasButton: UIButton!
+    
```

Comment:
Why are you using a button???? a button is never the solution on a cell, you should use the delegates like in tableviews